src/modules/menu/components/settings/MenuSettings.vue

```diff
@@ -1,6 +1,13 @@
 <script setup>
+import { computed } from 'vue'
+import { useStore } from 'vuex'
+
 import LangSwitchToolbar from '@/modules/i18n/components/LangSwitchToolbar.vue'
+import MapFooterAppVersion from '@/modules/map/components/footer/MapFooterAppVersion.vue'
```

Comment:
Not a good idea to do cross module import !

Reply:
I moved it to shared components